railgun/Makefile

```
@@ -1,4 +1,4 @@
-
+KIND_CLUSTER_NAME ?= "integration-tests"
```

This seems like something we'd want to default rather than setting directly in the target. There's no way to turn it off otherwise:
```suggestion
KIND_CLUSTER_NAME ?= "integration-tests"
KIND_KEEP_CLUSTER ?= "true"
```

---

If we add the `KIND_KEEP_CLUSTER` ENV override here, then the tests wont clean themselves up. The new `make test.integration.cluster` will generate the cluster without tearing it down, but other directive's default **should** be to tear it down.

---

Ah, okay, diff blinders on my part. After looking at the complete file, what you said makes sense, and my suggestion should indeed not go in.